docs/source/reference/numpysupported.rst

```diff
@@ -210,7 +210,7 @@ The following methods of Numpy arrays are supported:
 * :meth:`~numpy.ndarray.repeat` (no axis argument)
 * :meth:`~numpy.ndarray.reshape` (only the 1-argument form)
 * :meth:`~numpy.ndarray.sort` (without arguments)
-* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument)
+* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument. ``axis`` only supports ``integer`` values)
```

Comment:
As noted here https://github.com/numba/numba/pull/4397#discussion_r310565569, this line is now too long ( >80 chars), please could you wrap it, thanks.